api/http/handler/teams/team_create.go

```diff
@@ -62,5 +64,20 @@ func (handler *Handler) teamCreate(w http.ResponseWriter, r *http.Request) *http
 		return &httperror.HandlerError{http.StatusInternalServerError, "Unable to persist the team inside the database", err}
 	}
 
+	if payload.TeamLeaders != nil {
```

Comment:
pretty sure `range` will eliminate the need to do the `nil` check up front, so we should be safe taking this check out to reduce nesting.

Reply:
done